cdap-app-fabric/src/main/java/io/cdap/cdap/gateway/handlers/ProgramLifecycleHttpHandler.java

```diff
@@ -389,12 +394,19 @@ private void doPerformAction(FullHttpRequest request, HttpResponder responder, S
     ProgramType programType = getProgramType(type);
     ProgramId program = applicationId.program(programType, programId);
     Map<String, String> args = decodeArguments(request);
+    String latestVersion = getLatestAppVersion(new NamespaceId(namespaceId), appId);
```

Comment:
I should have done it in the first place as well... Pls update java doc for the new behavior

Reply:
added javadoc